distrib/docs/english/source/avisynthdoc/corefilters/version.rst

```diff
@@ -2,19 +2,59 @@
 Version
 =======
 
+**Version** by default generates a 24\ `fps`_, 10-second, `RGB24`_\-format video
+clip with a version, build time, copyright statement, and homepage information
+in 24-point, Arial, light yellow (:doc:`$ECF2BF <../syntax/syntax_colors>`) text
```

Comment:
The font is actually specific to Windows' defaults.  The other OSes use Terminus (this same output can be seen on Windows only if you've used the `NO_WIN_GDI` flag while building AviSynth+).  You can see this in the example images [here](https://i.imgur.com/OCNAx9K.jpg) and [here](https://i.redd.it/ms44fdlg50b61.png).

Reply:
Yes, I noticed that fonts are different for non-Windows OSs when I was documenting the Text filter and was looking at the source (the reason I asked pinterf if he can add a global flag on windows to force the Text filter over Subtitle). I don't have a system to test that on so right now I'm just going by what is in the wiki and sometimes adding, removing, rewording a few things for clarity. I will probably come back to some of these pages again sometime in the future and add anything that is missing. Of course, if you want to add info for other OSs be my guess :)

Reply:
On Windows default font for MessageClip is "Arial".
As for outputting the embedded fixed fonts instead of WinGDI TrueType, next time I can build next time a NO_WIN_GDI version as well.

Reply:
> On Windows default font for MessageClip is "Arial". As for outputting the embedded fixed fonts instead of WinGDI TrueType, next time I can build next time a NO_WIN_GDI version as well.

Sounds good, that would be helpful.

Reply:
test12 updated with x64_no_win_gdi version
https://drive.google.com/uc?export=download&id=1v2v8mDY5fHfBWUXvTmTMZLJC2nP1U6og

Reply:
Thank you!